static/app/views/dashboardsV2/controls.tsx

```diff
@@ -67,6 +67,7 @@ class Controls extends React.Component<Props> {
             priority="danger"
             message={t('Are you sure you want to delete this dashboard?')}
             onConfirm={onDelete}
+            disabled={!(dashboards.length > 1)}
```

Comment:
@shruthilayaj We also may need to prevent deleting the last dashboard for the API endpoint.

Reply:
Nit: Can we simplify the negation here and just do dashboards.length <= 1?

Reply:
lol sure thing